setup.py

```diff
@@ -1,53 +1,27 @@
+#!/usr/bin/env python
+
 # this script allows us to publish sureal in PyPi and the use it using 'pip install sureal'
 # run 'python setup.py sdist bdist_wheel' to create the distribution files within sureal/python/src/
 
-import os
-import setuptools
-from numpy.distutils.core import setup
-
-PACKAGE_NAME = 'sureal'
-
-parent_dir = os.path.dirname(__file__)
-
-
-def _version():
-    """ Get the local package version.
-    """
-    path = os.path.join(PACKAGE_NAME, "version.py")
-    namespace = {}
-    with open(path) as stream:
-        exec(stream.read(), namespace)
-    return namespace["__version__"]
-
-
-with open(os.path.join(parent_dir, "README.md"), "r") as fh:
-    long_description = fh.read()
+from setuptools import setup
 
-try:
-    import pypandoc
-    long_description = pypandoc.convert_text(long_description, 'rst', format='md')
```

Comment:
This conversion at runtime is not necessary, I just converted the README to rst here.

Reply:
I prefer writing Markdown, that's why that was included at the time. But fine by me.

Reply:
I'm with you, markdown is simpler to write. Took me a while to learn `rst`, but once you do, it's somewhat more readable... albeit still a bit complex to write at times.

pypi is all about `rst`, their markdown rendering is not working yet, even though one can technically now provie an `md`. With setupmeta, I try to support both (as per the doc), but even if you do, it doesn't quite render well yet.

You can see https://pypi.org/project/sureal/ -> not rendering well at all currently (will render well with this PR). 

You can see what I mean by "setupmeta tries to support both" by running `setup.py explain` on a setupmeta enabled project, you'll see this:
```
             long_description: (README.rst) SUREAL - Subjective quality ...
long_description_content_type: (README.rst) text/x-rst
```

If you use an `.md`, you should see that "content type" be `text/markdown`, but yet last time i tried to publish such a project, it wasn't rendering well...

Reply:
Another way to see how this metadata ends up being packaged is `grep Description sureal.egg-info/PKG-INFO`:
```
Description: SUREAL - Subjective quality scores recovery from noisy measurements
Description-Content-Type: text/x-rst
```